compiler/codegen/OMRAheadOfTimeCompile.hpp

```diff
@@ -40,28 +40,32 @@ class TR_Debug;
 namespace TR { class ExternalRelocation; }
 namespace TR { class IteratedExternalRelocation; }
 namespace TR { class AheadOfTimeCompile; }
-
+namespace TR { class RelocationRecord; }
+namespace TR { class RelocationRuntime;}
 namespace OMR
 {
 
 class OMR_EXTENSIBLE AheadOfTimeCompile
    {
-   public:
+public:
 
    TR_ALLOC(TR_Memory::AheadOfTimeCompile)
 
    AheadOfTimeCompile(uint32_t *headerSizeMap, TR::Compilation * c)
       : _comp(c), _sizeOfAOTRelocations(0),
-        _relocationData(NULL),
+        _relocationData(NULL)
+      #ifndef OMR_RELOCATION_RUNTIME
+      ,
         _aotRelocationKindToHeaderSizeMap(headerSizeMap)
+      #endif
```

Comment:
I don't think you need this ifdef anymore. You've already changed the code to not look at the table in OMR; it'll be an unused field but it'll be cleaner code that way.

Reply:
fixed, thank you for the comments!